README.md

```diff
@@ -1,57 +1,22 @@
-# Northcoders News API
+# NC News API
 
-## Background
+## To successfully connect to the two databases locally:
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
 
-## Kanban
 
-### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
+Your clone must include the .env files containing PGDATABASE=(nc_news/ nc_news_test)
```

Comment:
naturally, in production you'd want to protect your db names, but for the purposes of this project, this is probably ok

Reply:
Thanks alex, noted